kratos/expression/variable_expression_data_io.cpp

```diff
@@ -184,6 +374,29 @@ void VariableExpressionDataIO<Vector>::Assign(
     }
 }
 
+template<>
+void VariableExpressionDataIO<std::vector<Vector>>::Assign(
+    std::vector<Vector>& rOutput,
+    const Expression& rExpression,
+    const IndexType EntityIndex) const
+{
+    const auto& r_shape = rExpression.GetItemShape();
```

Comment:
check for degenerate item shapes

Reply:
as mentioned in the previous comment, added a full debug error and a comment.